### PR TITLE
Add share to home

### PR DESCRIPTION
**Pull request by loverajoel**
- Add new footer
- Add share

### CODE REVIEW

File: public/twitter.svg [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="90px" height="90px" viewBox="0 0 90 90" style="enable-background:new 0 0 90 90;" xml:space="preserve" fill="#fff">
+<g>
+	<path id="Twitter" d="M67.261,67.496H45.054c-3.087,0-5.712-1.08-7.869-3.25c-2.167-2.172-3.238-4.797-3.238-7.898v-7.904H65.59
+		c2.854,0,5.312-1.025,7.354-3.062c2.041-2.054,3.066-4.509,3.066-7.366c0-2.867-1.025-5.319-3.072-7.366
+		c-2.049-2.042-4.515-3.066-7.381-3.066H33.946V11.254c0-3.09-1.102-5.735-3.29-7.939C28.478,1.107,25.842,0,22.782,0
+		c-3.146,0-5.825,1.091-8.004,3.25C12.591,5.416,11.5,8.084,11.5,11.264v45.089c0,9.274,3.278,17.197,9.837,23.773
+		C27.901,86.715,35.814,90,45.066,90h22.203c3.082,0,5.729-1.107,7.93-3.314c2.203-2.197,3.302-4.849,3.302-7.936
+		c0-3.088-1.099-5.734-3.302-7.941C72.997,68.607,70.347,67.496,67.261,67.496z"/>
+</g>
```
Comment (Cosmitar) [inline public/twitter.svg:13]: What the heck?

File: src/App.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ import ResultHeader from './components/results/ResultHeader';
 import ResultList from './components/results/ResultList';
 import Footer from './components/footer/Footer';
 import Header from './components/header/Header';
+import Share from './components/share';
```
Comment (Cosmitar) [inline src/App.js]: Components filename should be named as the component, starting with capital letter -> ./components/share/Share.js

File: src/components/share/index.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    render() {
+        return (<div className='share-wrapper'>
+                    <span>Share</span>
+                    {this._tw()}
+                    {this._fb()}
+                </div>
+            );
+    }
+
+    _tw() {
```
Comment (Cosmitar) [inline src/components/share/index.js]: Url and function name are switched. _tw calls facebook sharer and _fb calls twitter intent XD

**Comment by Cosmitar** [comment]
:shipit: after :beer:

File: public/twitter.svg (hunk 1 above)
Comment (loverajoel) [inline public/twitter.svg:13]: we can chat about it

**Pull request merged**